Antonio_Ramos_Documentation

```diff
@@ -0,0 +1,171 @@
+
+- after working with ICT spyder4 was not opening, so the files located at "C:\Users\YOUR_USER_NAME\.spyder-py3\"   " C:\Users\YOUR_USER_NAME\.ipython" "C:\Users\YOUR_USER_NAME\.matplotlib"   were all deleted with copies placed in "C:\Users\aramosri\documents"
+
+- spyder 4 now opens correctly
+
+- upon opening spyder i was asked if I wanted to install "Kite" which "provides better and more accurate code completion in its editor when using packages like matplotlib numpy and pandas (Which are all being used in our python scripts) -- will ask prof buzard if she believes we should install
+
+- again trying to see if i can get "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\K-function Local_3Stage Trial" script to run past the importation of modules, decided to begin by downloading the packages as prompted at the top of the script
+
+- attemped to download package "plotnine" with command "conda install -c conda-forge plotnine"
+
+- install successful, package downloaded to C:\Apps-SU\Anaconda3\pkgs
+
+- attempted to download package "geopandas" with command "conda install -c conda-forge geopandas"
+
+-when solving the environment for the download of package geopandas the following message was outputted
+"
+Solving environment: failed with initial frozen solve. Retrying with flexible solve.
+Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
+Collecting package metadata (repodata.json): done
+Solving environment: done
+"
+
+- package "geopandas" installed to C:\Apps-SU\Anaconda3\pkgs
+
+- attempted to run "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\K-function Local_3Stage Trial" and was able to complete the process all modules imported, simulations completed, and shapefiles saved
+
+-attempting to open shapefiles with the following comand in spyder "gpd.read_file(G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\SHAPEFILES\POPULATION_POPULATION_Points.shp)"
+
+-invalid syntax error returned
+
+- attemped the same thing by running the following code
+"
+import geopandas as gpd 
+import matplotlib.pyplot as plt
+gdf = gpd.read_file(G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\SHAPEFILES\POPULATION_POPULATION_Points.shp)
+print(gdf.shape)
+print(gdf.head())
+gdf.plot()
+plt.show()
+"
+- again returned invalid syntax error 
+
+- found syntax error and corrected code, adding ' to each end of the file name G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\SHAPEFILES\POPULATION_POPULATION_Points.shp resulting in the following code
+"
+import geopandas as gpd 
+import matplotlib.pyplot as plt
+gdf = gpd.read_file('G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\SHAPEFILES\POPULATION_POPULATION_Points.shp')
+print(gdf.shape)
+print(gdf.head())
+gdf.plot()
+plt.show()
+"
+- code ran succesfully and I was able to see the scatter plot diagram
+
+(Skip to line 143)
```

Comment:
Please put a little more detail here. At first, I didn't know whether you meant line 143 of this documentation file or to go looking into the python script for that line. A sentence or two explaining why the next chunk is not relevant should clear this up.

Reply:
Will do.